README.md

```diff
@@ -95,6 +93,7 @@ higher-level integrations with UI frameworks.
 
 ## Resources
 
+* [droidcon NYC 2019 - Square Workflow](https://docs.google.com/presentation/d/19-DkVCn-XawssyHQ_cboIX_s-Lf6rNg-ryAehA9xBVs/edit#slide=id.g4088c7565d_2_5)
```

Comment:
Updated the title to match the other droidcon links and added "video coming soon":
```suggestion
* Square Workflow – Droidcon NYC 2019 ([slides](https://docs.google.com/presentation/d/19-DkVCn-XawssyHQ_cboIX_s-Lf6rNg-ryAehA9xBVs/edit#slide=id.g4088c7565d_2_5), video coming soon…)
```